etc/kayobe/hooks/overcloud-service-deploy/post.d/deploy-os-capacity-exporter.yml

```diff
@@ -0,0 +1 @@
+../../../ansible/deploy-os-capacity-exporter.yml
```

Comment:
I'm a bit on the fence about whether we should include this hook by default, rather than document how to add the hook. Any other thoughts?

Reply:
Right now the playbook should skip all tasks (with the exception of os_exporter removal) so it was more for ease of deployment. 

Perhaps the hook should exist by default if os-capacity is part of the standard deployment, or document adding it if we don't include it for now?

Reply:
Discussing [here](https://stackhpc.slack.com/archives/C044YBYMJBZ/p1709635641493309)

Reply:
Agreed to enable by default.